src/angularComponent/ngcOmniboxComponent.js

```diff
@@ -58,6 +58,11 @@ export default {
     hideOnBlur: '@',
     isSelectable: '&',
     canShowSuggestions: '&',
-    requireMatch: '<?'
+    requireMatch: '<?',
+    onChosen: '&',
+    onUnchosen: '&',
+    onNgModelChange: '&',
+    onSuggestionsShown: '&',
```

Comment:
I prefer `onShowSuggestions` and `onHideSuggestions` for the following reasons:

1. It's shorter.
2. A verb follows the word "on".

Not going to hold my approval back on this point though.